apps/browser/lib/api/utils/subsidy.ts

```diff
@@ -3,7 +3,7 @@ import { SupabaseClient } from "./supabase";
 type Subsidy = "completions" | "embedding";
 
 const SUBSIDY_CAP: Record<Subsidy, number> = {
-  "completions": 50,
```

Comment:
Can we leave this at `50`? I worry that some goals may not be able to be completed with a lower completions cap.